webapp/src/main/webapp/js/brandingColors.js

```diff
@@ -0,0 +1,394 @@
+/* $This file is distributed under the terms of the license in LICENSE$ */
+
+
+let theme = "themes/wilma/";
+
+const colorNameConvertor = {
+    "primary-lighter": "themePrimaryColorLighter",
+    "primary-base": "themePrimaryColor",
+    "primary-darker": "themePrimaryColorDarker",
+    "banner-base": "themeBannerColor",
+    "secondary-base": "themeSecondaryColor",
+    "accent-base": "themeAccentColor",
+    "text-base": "themeTextColor",
+    "link-base": "themeLinkColor"
+}
+
+const colorNameConvertorInverse = Object.keys(colorNameConvertor).reduce((acc, key) => {
+        acc[colorNameConvertor[key]] = key;
+        return acc;
+    }, {})
+
+
+const defaultColorPalete = {
+    "themes/wilma/": {
```

Comment:
Hardcoding theme names/paths is not an option. It will be broken once theme is renamed for customization.